.github/workflows/ci-osx.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Set up env
         shell: "bash -l {0}"
         run: |
-          conda create -n env python==${{matrix.python-version}} wheel pip compilers 'clang>=12.0.1'
+          conda create -n env python=${{matrix.python-version}} wheel pip compilers 'clang>=12.0.1'
```

Comment:
Having `==`s effectively means `3.10.0` instead of `3.10.0`+. As we likely don't want to pin patch releases to `0`, relaxing that here. May be part of the issue we were seeing as well

Reply:
Oh wow. TIL. :+1: